diff -urN abrt-1.1.13.orig/lib/Utils/DebugDump.cpp abrt-1.1.13/lib/Utils/DebugDump.cpp
--- abrt-1.1.13.orig/lib/Utils/DebugDump.cpp	2010-11-08 13:31:13.897933729 +0300
+++ abrt-1.1.13/lib/Utils/DebugDump.cpp	2010-11-08 13:32:09.596932508 +0300
@@ -330,7 +330,7 @@
         SaveText(FILENAME_KERNEL, buf.release);
         SaveText(FILENAME_ARCHITECTURE, buf.machine);
         std::string release;
-        LoadTextFile("/etc/redhat-release", release);
+        LoadTextFile("/etc/fedora-release", release);
         const char *release_ptr = release.c_str();
         unsigned len_1st_str = strchrnul(release_ptr, '\n') - release_ptr;
         release.erase(len_1st_str); /* usually simply removes trailing '\n' */
