commit 306a4686200c33e34c0650b6ad09b1a5a3f32a77
Author: Jiri Moskovcak <jmoskovc@redhat.com>
Date:   Tue Apr 13 16:51:30 2010 +0200

    fixed problem with localized yum output rhbz#581804

diff --git a/src/Daemon/abrt-debuginfo-install b/src/Daemon/abrt-debuginfo-install
index 521d42a..84d2446 100755
--- a/src/Daemon/abrt-debuginfo-install
+++ b/src/Daemon/abrt-debuginfo-install
@@ -303,7 +303,7 @@ $debug && echo "build_ids:$build_ids"
 # When we look for debuginfo we need only -debuginfo* repos, we can disable the rest
 # and thus make it faster.
 yum_repo_opts="'--disablerepo=*'"
-for enabled_repo in `yum repolist all | grep 'enabled:' | cut -f1 -d' ' | grep -v -- '-debuginfo'`; do
+for enabled_repo in `LANG=C yum repolist all | grep 'enabled:' | cut -f1 -d' ' | grep -v -- '-debuginfo'`; do
     yum_repo_opts="$yum_repo_opts '--enablerepo=${enabled_repo}-debuginfo*'"
 done
 
