--- abrt-1.0.9/src/Gui/ccgui.glade	2010-03-31 10:34:14.000000000 +0200
+++ abrt-1.0.9_hideprefs/src/Gui/ccgui.glade	2010-03-31 21:39:27.653364662 +0200
@@ -102,7 +102,7 @@
                     <child>
                       <widget class="GtkImageMenuItem" id="miPreferences">
                         <property name="label">gtk-preferences</property>
-                        <property name="visible">True</property>
+                        <property name="visible">False</property>
                         <property name="use_underline">True</property>
                         <property name="use_stock">True</property>
                         <property name="always_show_image">True</property>
