diff -ru abrt-0.0.1.orig/lib/MiddleWare/RPMInfo.cpp abrt-0.0.1/lib/MiddleWare/RPMInfo.cpp
--- abrt-0.0.1.orig/lib/MiddleWare/RPMInfo.cpp	2009-03-03 16:17:40.000000000 +0000
+++ abrt-0.0.1/lib/MiddleWare/RPMInfo.cpp	2009-03-08 15:18:54.000000000 +0000
@@ -72,7 +72,7 @@
     Header header;
     if ((header = rpmdbNextIterator(iter)) != NULL)
     {
-        rpmfi fi = rpmfiNew(ts, header, RPMTAG_BASENAMES, 0);
+        rpmfi fi = rpmfiNew(ts, header, RPMTAG_BASENAMES, RPMFI_NOHEADER);
         pgpHashAlgo hashAlgo;
         std::string headerHash;
         char computedHash[1024] = "";
