--- abrt-1.1.1/src/Daemon/abrt.conf	2010-05-12 15:48:18.000000000 +0200
+++ abrt-1.1.1_/src/Daemon/abrt.conf	2010-07-19 21:07:14.228273198 +0200
@@ -5,7 +5,7 @@
 # in the file gpg_keys
 OpenGPGCheck = yes
 # Blacklisted packages
-BlackList = nspluginwrapper, valgrind, strace
+BlackList = nspluginwrapper, valgrind, strace, mono-core
 # Process crashes in executables which do not belong to any package?
 ProcessUnpackaged = no
 # Blacklisted executable paths (shell patterns)
