commit 466f32513e67bf937d42321d1b979c5a477905b6
Author: Jiri Moskovcak <jmoskovc@redhat.com>
Date:   Mon Mar 28 17:26:00 2011 +0200

    applet: fixed the notification program name

diff --git a/src/applet/applet_gtk.c b/src/applet/applet_gtk.c
index 0ba9499..8071ec1 100644
--- a/src/applet/applet_gtk.c
+++ b/src/applet/applet_gtk.c
@@ -355,7 +355,7 @@ struct applet *applet_new(const char* app_name)
         applet->ap_menu = create_menu(applet);
     }
 
-    notify_init(app_name);
+    notify_init("abrt");
     return applet;
 }
 
